app/presenters/hyrax/version_list_presenter.rb

```diff
@@ -19,13 +19,10 @@ def initialize(service)
     # @return [Hyrax::VersionListPresenter] an enumerable of presenters for the
     #   relevant file versions.
     #
-    # @raise [ArgumentError] if we can't build an enu
-    def self.for(file_set:)
-      original_file = if file_set.respond_to?(:original_file)
```

Comment:
I believe this `if` was here for ActiveFedora compatibility—`FileSetFileService` probably only works in Valkyrie?

Reply:
ah, i removed it because *all* model classes for FileSet have `#original_file`, so it prevents the service from ever being reached. 

doing a class based switch statement instead.